src/codecoverage/backend/requirements-dev.txt

```diff
@@ -2,3 +2,6 @@
 
 responses
 aresponses
+# pytest-asyncio is a dependency of aresponses, but source tarball does not
```

Comment:
Can you file an issue upstream?

Reply:
already is https://github.com/pytest-dev/pytest-asyncio/issues/94